### ISSUE TITLE
scripts/go-test is not removed with make cleanall

### ISSUE THREAD

**Issue by mem** [apptainer/singularity#3980]
## Version of Singularity:

edfc9d7e0d864871357523547eb0e85919d2bab5

### Expected behavior

scripts/go-test is removed when you run make cleanall

### Actual behavior

scripts/go-test is not removed when you run make cleanall

### Steps to reproduce behavior

```
./mconfig
make -C builddir $PWD/scripts/go-test
test -e scripts/go-test && echo it exists
make -C builddir cleanall
test -e scripts/go-test && echo it exists
```

that should print "it exists" only once.

**Comment by carterpeel** [comment]
Hello,

This is a templated response that is being sent out to all open issues.  We are working hard on 'rebuilding' the Singularity community, and a major task on the agenda is finding out what issues are still outstanding.


**Please consider the following:**

1. Is this issue a duplicate, or has it been fixed/implemented since being added?
2. Is the issue still relevant to the current state of Singularity's functionality?
3. Would you like to continue discussing this issue or feature request?


Thanks,
Carter

**Comment by pedroalvesbatista** [comment]
@mem Still getting this problem ? Have this been solved already ? If yes, what work around have you followed and applied ?

We're looking into the issue carefully, soon will bring to community and discuss ways to better solve as well address this. Thankyou for keeping the interest in the subject.

**Comment by kmuriki** [comment]
This repository is no longer active. Please open new issue at https://github.com/apptainer/apptainer.
